docs/reference/index.md

```diff
@@ -258,6 +258,55 @@ As a convenience for production deployments, with the below environment variable
 CS_DATABASE__INSTALL_AWS_RDS_CERT_BUNDLE="true"
 ```
 
+## Multitenant
+
+CipherStash Proxy supports multitenant applications by allowing clients to switch between different keysets at runtime. This enables a single Proxy instance to handle encrypted data for multiple tenants, with each tenant's data protected by separate encryption keys.
+
+### Keyset Commands
```

Comment:
It may be helpful to the users if we tell them whether they need to set both or one of those